src/main/java/uk/gov/hmcts/cmc/claimstore/rpa/RequestForJudgementNotificationService.java

```diff
@@ -47,12 +47,10 @@ public void notifyRobotics(CountyCourtJudgmentEvent event) {
         CountyCourtJudgmentType countyCourtJudgmentType = event.getClaim().getCountyCourtJudgment().getCcjType();
         switch (countyCourtJudgmentType) {
             case DEFAULT:
-                EmailData emailData = prepareEmailData(event.getClaim());
-                emailService.sendEmail(emailProperties.getSender(), emailData);
-                break;
             case DETERMINATION:
             case ADMISSIONS:
-                //No RPA email sent
+                EmailData emailData = prepareEmailData(event.getClaim());
```

Comment:
As per story we should not send email for determination but this will send email for "determination" as well.

Reply:
Fixed